index.html

```diff
@@ -2429,12 +2413,6 @@ <h2>Chunk ordering</h2>
 and <a href="#idat-image-data"><span class="chunk">IDAT</span></a> </td>
 </tr>
 
-<tr>
-<td class="Regular"><a href="#11cICP"><span class="chunk">cICP</span></a> </td>
-<td class="Regular">No</td>
-<td class="Regular">Before <a href="#plte-palette"><span class="chunk">PLTE</span></a>
-and <a href="#idat-image-data"><span class="chunk">IDAT</span></a> </td>
-</tr>
```

Comment:
Not sure these were supposed to be removed?

Reply:
I didn't think removing them was a great idea, but [you said to put them in a separate PR](https://github.com/w3c/PNG-spec/pull/105#discussion_r833748269) so I did.